### PR TITLE
Refine recipes for classification

### PR DESCRIPTION
**Pull request by harimkang**
### Summary

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=OneNote.File>
<meta name=Generator content="Microsoft OneNote 15">
</head>

<body lang=en-US style='font-family:Calibri;font-size:11.0pt'>

<div style='direction:ltr'>

  |   |   | Small |   | Medium |   | Large |  
-- | -- | -- | -- | -- | -- | -- | -- | --
Task | Model | otx_version | Test/Accuracy | Train/e2e_time | Test/Accuracy | Train/e2e_time | Test/Accuracy | Train/e2e_time
MULTI_CLASS_CLS | deit_tiny | develop | 1.0 | 19 | 0.7831 | 38 | 0.8385 | 436
  |   | This PR | 1.0 | 11 | 0.7998 | 89 | 0.8540 | 263
  | dino_v2 | develop | 0.9842 | 40 | 0.8435 | 206 | 0.8960 | 883
  |   | This PR | 1.0 | 15 | 0.8428 | 245 | 0.8972 | 847
  | efficientnet_b0 | develop | 0.9989 | 19 | 0.7704 | 116 | 0.8338 | 288
  |   | This PR | 1.0 | 10 | 0.7819 | 88 | 0.8338 | 296
  | efficientnet_v2 | develop | 0.9966 | 26 | 0.8525 | 123 | 0.8854 | 434
  |   | This PR | 1.0 | 17 | 0.8577 | 148 | 0.8878 | 564
  | mobilenet_v3_large | develop | 0.9842 | 18 | 0.7733 | 105 | 0.8246 | 401
  |   | This PR | 0.9831 | 10 | 0.7889 | 63 | 0.8263 | 265

</div>

</body>

</html>

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have ran e2e tests and there is no issues.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: src/otx/recipe/classification/multi_class_cls/efficientnet_b0.yaml [hunk 1]
```diff
@@ -63,39 +56,3 @@ overrides:
...
-            scale: $(input_size)
-            is_numpy_to_tvtensor: true
-        - class_path: torchvision.transforms.v2.ToDtype
-          init_args:
-            dtype: ${as_torch_dtype:torch.float32}
-            scale: false
-        - class_path: torchvision.transforms.v2.Normalize
-          init_args:
-            mean: [123.675, 116.28, 103.53]
-            std: [58.395, 57.12, 57.375]
```
Comment (sungchul1) [inline src/otx/recipe/classification/multi_class_cls/efficientnet_b0.yaml:101]: `reset` should be updated.

File: src/otx/recipe/classification/multi_class_cls/efficientnet_v2.yaml [hunk 2]
```diff
@@ -62,39 +55,3 @@ overrides:
...
-            scale: $(input_size)
-            is_numpy_to_tvtensor: true
-        - class_path: torchvision.transforms.v2.ToDtype
-          init_args:
-            dtype: ${as_torch_dtype:torch.float32}
-            scale: false
-        - class_path: torchvision.transforms.v2.Normalize
-          init_args:
-            mean: [123.675, 116.28, 103.53]
-            std: [58.395, 57.12, 57.375]
```
Comment (sungchul1) [inline src/otx/recipe/classification/multi_class_cls/efficientnet_v2.yaml:100]: `reset` should be updated.

**Comment by sungchul1** [comment]
Actually, it is no problem for validation and test transforms in the above comments because otx ignores if there are no override components although those keys are set in `reset`.
But I think it should be updated because it is the example that users can refer to for their own recipes.

File: src/otx/recipe/classification/multi_class_cls/efficientnet_b0.yaml (hunk 1 above)
Comment (harimkang) [reply src/otx/recipe/classification/multi_class_cls/efficientnet_b0.yaml:101]: Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/3776/commits/8bbe02cd9116631d163bbd4a80bf3b1f7eae1bea

File: src/otx/recipe/classification/multi_class_cls/efficientnet_v2.yaml (hunk 2 above)
Comment (harimkang) [reply src/otx/recipe/classification/multi_class_cls/efficientnet_v2.yaml:100]: Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/3776/commits/8bbe02cd9116631d163bbd4a80bf3b1f7eae1bea

**Review by sungchul1 — APPROVED** [review]

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**